src/utils/warnings.ts

```diff
@@ -0,0 +1,11 @@
+import { Strategy } from '../types';
+
```

Comment:
I think this was already added to a separate file in `https://github.com/yearn/yearn-watch/pull/193` so we should remove this file